lib/routes/account.js

```diff
@@ -406,52 +412,55 @@ module.exports = function (
           .then(sendVerifyLoginEmail)
           .then(recordSecurityEvent)
           .then(createResponse)
+          .catch(gateSigninUnblock)
           .done(reply, reply)
 
+        function checkUnblockCode (e) {
+          // only set `unblockCode` if it is required from customs
+          unblockCode = request.payload.unblockCode
+          if (e.verificationReason === 'login' && unblockCode) {
```

Comment:
It looks like `e.verificationReason` needs to be `e.output.payload.verificationReason`, all the "extra" data passed to [AppError is put into this.output.payload](https://github.com/mozilla/fxa-auth-server/blob/cff42f7f1c6301bd00890a518eeebad92109e63d/lib/error.js#L71:L74).